src/js/components/ac-drawer.js

```diff
@@ -5,13 +5,17 @@ import { AppBar, MenuItem, LeftNav, FontIcon } from 'material-ui';
 import { Link } from 'react-router';
 /*eslint-enable no-unused-vars*/
 
+import { connect } from 'react-redux';
```

Comment:
components should not connect to redux... I understand the appeal for doing so in this component, but the action should be passed down from the containers (which do connect redux).